Makefile

```diff
@@ -2,11 +2,13 @@ init:
 	pip install -r requirements.txt
 devel:
 	pip install -r requirements_dev.txt
-	pre-commit install
+	pre-commit install --hook-type pre-push --install-hooks -t post-checkout -t post-merge
+	curl -sSL https://install.python-poetry.org | python3 -

```

Comment:
What is this curl for? You should be able to just `pip install poetry`?

Reply:
huh you know what you're right. I was looking at the docs for installation and the first one option was [`pipx`](https://python-poetry.org/docs/#installing-with-pipx).

The next one was the curl cmd but now i see under the [`advanced`](https://python-poetry.org/docs/#installing-manually) section you can just use pip. I'll make the change.

Reply:
see latest commits